requirements-chrouter.in

```diff
@@ -0,0 +1,14 @@
+-e git+https://github.com/assembl/sockjs-tornado.git@conflicting_localhost#egg=sockjs_tornado
+tornado==4.3
+
+# Those are required, but present in requirements.in.
+
+# PyJWT==1.4.0
+# pytz==2016.4
+# pyzmq==15.2.0
+# raven==5.21.0
+# requests==2.10.0
+# simplejson==3.8.2
```

Comment:
In the case of overlaps, what happens? In case of overlapping packages with different package versions, do we append both to the frozen pip package?

Reply:
pip-tool would complain about any conflict. I uncommented those, because a docker-based changes-router would need them.